maestro/pom.xml

```diff
@@ -13,9 +13,18 @@
 
 
     <dependencies>
+        <!-- https://mvnrepository.com/artifact/org.mockito/mockito-junit-jupiter -->
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-junit-jupiter</artifactId>
+            <version>3.9.0</version>
```

Comment:
maybe this version should go into the root dependency management. to make sure we dont have different versions in the different modules

Reply:
I agree. I'll move it to the root dependency management.